sotodlib/mapmaking/ml_mapmaker.py

```diff
@@ -176,12 +186,12 @@ class SignalMap(Signal):
     """Signal describing a non-distributed sky map."""
     def __init__(self, shape, wcs, comm, comps="TQU", name="sky", ofmt="{name}", output=True,
             ext="fits", dtype=np.float32, sys=None, recenter=None, tile_shape=(500,500), tiled=False,
-            interpol=None):
+            interpol=None, glitch_flags: str = "flags.glitch_flags"):
         """Signal describing a sky map in the coordinate system given by "sys", which defaults
         to equatorial coordinates. If tiled==True, then this will be a distributed map with
         the given tile_shape, otherwise it will be a plain enmap. interpol controls the
         pointing matrix interpolation mode. See so3g's Projectionist docstring for details."""
-        Signal.__init__(self, name, ofmt, output, ext)
+        Signal.__init__(self, name, ofmt, output, ext, glitch_flags)
```

Comment:
Nothing is done with glitch_flags (even in the Signal constructor).  You should cache it in self.glitch_flags.

My inclination is for that to be done, independently, in the SignalMap and SignalCut classes, rather than through the base Signal class.  I think one can imagine signals that don't care about glitch_flags / tod stuff at all (?).